buildlib/azure-pipelines-release.yml

```diff
@@ -30,7 +30,22 @@ stages:
   - stage: GitHubDraft
     jobs:
       - job: DraftRelease
+        container: centos7
         steps:
+        - checkout: self
+          clean: true
+          path: "we/need/to/go/deeper"
+
+        - bash: ./autogen.sh
+          displayName: Setup autotools
+
+        - bash: |
+            set -eE
+            gcc --version
+            ./contrib/configure-release --with-java
```

Comment:
should be --without-java (need minimal configure for src rpm build)

Reply:
fixed